### DISCUSSION TITLE
If I want to use  6D array in "actor.odf_slicer", how can i do?

### DISCUSSION THREAD

**Discussion by zp1008611** [dipy/dipy#3036] [Scientific Visualization]
## Description
The dipy code of "actor.odf_slicer" is as following:
def odf_slicer(
    odfs,
    affine=None,
    mask=None,
    sphere=None,
    scale=0.5,
    norm=True,
    radial_scale=True,
    opacity=1.0,
    colormap=None,
    global_cm=False,
    B_matrix=None,
):
    """Create an actor for rendering a grid of ODFs given an array of
    spherical function (SF) or spherical harmonics (SH) coefficients.

    Parameters
    ----------
    odfs : ndarray
        4D ODFs array in SF or SH coefficients. If SH coefficients,
        `B_matrix` must be supplied.
        
        
If the dimension of odfs is 6D or 8D, how can i do?
Thanks

**Comment by skoudoro** [comment]
> If the dimension of odfs is 6D or 8D, how can i do?

I will reshape my data. What are your other dimensions ? currently, it is X, Y, Z, SH

**Comment by skoudoro** [comment]
also you can do 5D by adding a slicer and switching between volume with the slicer
